docs/workload-development-kit/workload-environment.md

```diff
@@ -37,6 +37,31 @@ The workloads can run in two environments: local and cloud. In local (devmode),
 
 ## Development environment
 
+### Workload NuGet package structure
```

Comment:
This entire section doesn't quite fit here. I suggest you move it as a stan alone after the end of the Development environment section. I.e. before Local development mode (devmode).